### ISSUE TITLE
Planning to deprecate old x-tab component

### ISSUE THREAD

**Issue by rajasegar** [rajasegar/ember-x-tabs#27]
I am planning to remove all the `x-tab` component working with element ids for associating tabs and panes. Instead thinking about making contextual component version `x-tabs` the default one.
Since the elementId implementation is having a lot of issues and not scalable for future ember versions, we can switch to contextual components completely.
For migration I can write a codemod if needed. 

@cah-briangantzler Thoughts?

**Comment by cah-briangantzler** [comment]
Yes, the old ones should be deprecated and the contextual ones be the default ones. I am fascinated with code mods but not near good enough to write one.

Dying to write a modifier version of this. Would work fine for the page control, but the tab control would be left hiding the DOM instead of removing. Dont think it would be wise to write a modifier for hald of it (just the page control)

**Comment by rajasegar** [comment]
Done with v2.0.0
